tools/should-skip-ci.js

```diff
@@ -0,0 +1,95 @@
+import { readFileSync } from "fs";
+
+const SKIP_CACTI = "skip-cacti-ci";
+const MaintainersFile = "MAINTAINERS.md";
+//regular expression to get the maintainers in MAINTAINERS.md
+const MAINTAINERS_REGEX =
+  /\|\s*([A-Za-z\s]+)\s*\|\s*\[([^\]]+)\]\[[^\]]+\]\s*\|\s*([A-Za-z0-9_-]+|-)*/g;
+
+const main = async () => {
+  const markdownContent = readFileSync(MaintainersFile, "utf-8");
+
+  const args = process.argv.slice(2);
+  const pullReqUrl = args[0];
+  const committerLogin = args[1];
+
+  //Uncomment these lines for local machine testing purposes:
+  //const pullReqUrl = "https://api.github.com/repos/zondervancalvez/cactus/pulls/7";
```

Comment:
Please change these references to PR's belonging to Cacti repo (not `zondervan` user repo) for ease of use? This is needed so that if one needs to quickly debug any PR which fails to do so, can just change the PR number at the end of this URL and test it out, instead of changing the origin name as well from `zondervan` to `hyperledger`